src/main/com/fulcrologic/guardrails/core.cljc

```diff
@@ -785,7 +785,17 @@
        {:arglists '([name doc-string? attr-map? [params*] gspec prepost-map? body?]
                     [name doc-string? attr-map? ([params*] gspec prepost-map? body?) + attr-map?])}
        [& forms]
-       (>defn* &env &form forms {:private true}))
+       (>defn* &env &form forms {:private? true}))
+
+     (comment
```

Comment:
I'd prefer not to leave comment blocks in the main source like this.  If you want to keep this code I'd prefer you put this in a test. Otherwise delete it.